.circleci/config.yml

```diff
@@ -1,4 +1,12 @@
 version: 2
+
+init: &init
+  run:
+    name: init
+    command: |
+      echo '. .circleci/shared.bash' >> "$BASH_ENV"
+      . .circleci/shared.bash
```

Comment:
Do we need both calls? I understand the first is how Circle loads those into its context, right?

Reply:
> Do we need both calls? 
I don't think so. 

>  I understand the first is how Circle loads those into its context, right?
yeap.

Removing.